### DISCUSSION TITLE
Include a tooltip on hover to display long track/album titles that are not fully displayed

### DISCUSSION THREAD

**Discussion by takaji10** [TT-ReBORN/Georgia-ReBORN#18] [Q&A] [answered]
Some of the classical music albums in my library have artist/album/track tags that are too long to be rendered in the playlist and playbar unless I stretch the size of foobar2000 more than I want. It would be great if the user could hover over the shortened text to see the full string.

Examples where it could be helpful:

![image](https://user-images.githubusercontent.com/1534470/125165434-257a1080-e197-11eb-8018-4d034968786a.png)

![image](https://user-images.githubusercontent.com/1534470/125165442-2ca11e80-e197-11eb-89b1-fa7e77f64619.png)

![image](https://user-images.githubusercontent.com/1534470/125165521-9a4d4a80-e197-11eb-804c-d543856246fe.png)

![image](https://user-images.githubusercontent.com/1534470/125165534-a76a3980-e197-11eb-8bee-efa059067d94.png)

**Comment by TT-ReBORN** [comment]
Good idea. Will add this on my ToDo list.

**Reply by takaji10** [reply]
Awesome, thank you!

One other note, you should follow the same convention you use for shortening track titles and also use "..." to show that an album title/list of artists is being shortened. Otherwise it looks like I've got an artist named "Polish" :P

**Reply by takaji10** [reply]
When you do implement this, it would also be helpful to have the tooltip state the selected playback order when hovering on the new playback order button - otherwise the user needs to guess the playback state based on the symbols.

**Comment by TT-ReBORN** [comment, marked as answer]
This new feature has been implemented.
